dart/lib/src/protocol/sentry_event.dart

```diff
@@ -88,23 +92,25 @@ class SentryEvent {
   ///
   /// It's `runtimeType` and `toString()` are logged.
   /// If it's an Error, with a stackTrace, the stackTrace is logged.
-  /// If this behavior is undesirable, consider using a custom formatted [message] instead.
+  /// If this behavior is undesirable, consider using a custom formatted
+  /// [message] instead.
   dynamic get throwable => (_throwable is ThrowableMechanism)
       ? (_throwable as ThrowableMechanism).throwable
       : _throwable;
 
-  /// A Throwable decorator that holds a Mechanism related to the decorated Throwable
+  /// A throwable decorator that holds a [Mechanism] related to the decorated
+  /// [throwable]
   ///
-  /// Use the 'throwable' directly if you don't want the decorated Throwable
+  /// Use the [throwable] directly if you don't want the decorated throwable
   dynamic get throwableMechanism => _throwable;
 
-  /// an optional attached StackTrace
-  /// used when event has no throwable or exception, see [SentryOptions.attachStacktrace]
-  final SentryStackTrace? stackTrace;
+  /// One or multiple chained (nested) exceptions that occurred in a program.
+  final List<SentryException>? exceptions;
```

Comment:
on java we have a class called SentryValues that encapsulates the list, https://github.com/getsentry/sentry-java/blob/93f3c221c8923ee6894d7f8edcaffb0a16cb6f14/sentry/src/main/java/io/sentry/SentryValues.java
not sure if its necessary as we don't use reflection for serialization, but worth saying it.

Reply:
Yeah, I also saw that. .NET has it, too, but with a comment saying it could be removed because they do serialization themselves, too. Sooo I didn't bother to implement it as we do serialization, too.